src/components/Settings/SettingsPlex.tsx

```diff
@@ -52,12 +62,143 @@ interface SyncStatus {
   libraries: Library[];
 }
 
+interface PresetServerDisplay {
+  name: string;
+  ssl: boolean;
+  uri: string;
+  address: string;
+  host: string;
+  port: number;
+  local: boolean;
+}
 interface SettingsPlexProps {
   onComplete?: () => void;
 }
 
+function PlexPresetOption(props: { key: string; server: PresetServerDisplay }) {
+  if (props.server.local) {
+    return (
+      <option key={props.server.uri} value={props.server.uri}>
+        {' '}
+        {props.server.name}
+        {' ('}
+        {props.server.address}
+        {') [local]'}
+      </option>
+    );
+  }
+  return (
+    <option key={props.key} value={props.server.uri}>
+      {' '}
+      {props.server.name}
+      {' ('}
+      {props.server.address}
+      {') [remote]'}
+    </option>
+  );
```

Comment:
This whole block isn't necessary. Drop the if statement above and do everything in one line.